### PR TITLE
chore: bump deps

### PR DESCRIPTION
**Pull request by DmitriyMV**
Bump kernel

Signed-off-by: Dmitriy Matrenichev <dmitry.matrenichev@siderolabs.com>

### CODE REVIEW

**Review by smira — APPROVED** [review]

File: Pkgfile [hunk 1]
```diff
@@ -7,9 +7,9 @@ vars:
 
   # upgrade once pahole > 1.24 is released, fix is here: https://git.kernel.org/pub/scm/devel/pahole/pahole.git/commit/?h=next&id=bcc648a10cbcd0b96b84ff7c737d56ce70f7b501
   # renovate: datasource=git-tags extractVersion=^binutils-(?<version>.*)$ versioning=loose depName=git://sourceware.org/git/binutils-gdb.git
-  binutils_version: 2_39
-  binutils_sha256: 645c25f563b8adc0a81dbd6a41cffbf4d37083a382e02d5d3df4f65c09516d00
-  binutils_sha512: 68e038f339a8c21faa19a57bbc447a51c817f47c2e06d740847c6e9cc3396c025d35d5369fa8c3f8b70414757c89f0e577939ddc0d70f283182504920f53b0a3
+  binutils_version: 2_40
```
Comment (frezbo) [inline Pkgfile:10]: pahole new version released? See the comment above

**Review by frezbo — CHANGES_REQUESTED** [review]
binutils cannot be updated until pahole 1.24

**Comment by DmitriyMV** [comment]
@frezbo it looks like 1.24 was released 7 months ago https://git.kernel.org/pub/scm/devel/pahole/pahole.git/tag/?h=v1.24

File: Pkgfile (hunk 1 above)
Comment (DmitriyMV) [reply Pkgfile]: Reverted.

**Comment by frezbo** [comment]
I guess > 1.24 :sweat_smile:

**Comment by DmitriyMV** [comment]
> I guess > 1.24 😅

Yeah, I reread the comment 🙂 and reverted. Only kernel bump now.

**Review by frezbo — APPROVED** [review]

**Pull request merged**